common/types/config.ts

```diff
@@ -6,8 +6,7 @@
 import { schema, TypeOf } from '@osd/config-schema';
 
 export const configSchema = schema.object({
-  // TODO: add here to prevent this plugin from being loaded
-  // enabled: schema.boolean({ defaultValue: true }),
+  enabled: schema.boolean({ defaultValue: true }),
```

Comment:
enable this will allow whole dashboard assistant been disabled, is this expected?

Reply:
Yes, I think it's upon agreement. Is it true? @ruanyl

Reply:
I think so, this flag allows us to turnoff the entire plugin